packages/react-ui-map/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rentpath/react-ui-map",
-  "version": "0.0.13",
+  "version": "0.0.14",
```

Comment:
Technically this is a minor version since you're introducing new features that are backwards compatible.  Please make this `0.1.0`